ArcGISRuntimeSDKQt_CppSamples/Layers/DisplayAnnotation/DisplayAnnotation.cpp

```diff
@@ -0,0 +1,76 @@
+// [WriteFile Name=DisplayAnnotation, Category=Layers]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifdef PCH_BUILD
+#include "pch.hpp"
+#endif // PCH_BUILD
+
+#include "DisplayAnnotation.h"
+
+#include "AnnotationLayer.h"
+#include "FeatureLayer.h"
+#include "Map.h"
+#include "MapQuickView.h"
+#include "ServiceFeatureTable.h"
+
+using namespace Esri::ArcGISRuntime;
+
+namespace
+{
+static const QUrl url("https://services1.arcgis.com/6677msI40mnLuuLr/arcgis/rest/services/East_Lothian_Rivers/FeatureServer/0");
```

Comment:
Be careful of the `static` keyword, it has different meanings in different contexts - some of which are C legacy! 

In this context the static keyword limits the value to file scope - exactly the same as an anonymous namespace, making the static keyword redundant!